### ISSUE TITLE
Issue certain commands after deploy hook was successful

### ISSUE THREAD

**Issue by RockyTV** [Neilpang/acme.sh#1129]
I don't know if this is possible yet, but could there be a config that could work just like the `installcmd` but will only run when the specified deploy hooks were successful?

**Comment by Neilpang** [comment]
If the deploy hook is written by you, you can issue the command in it.

**Comment by RockyTV** [comment]
The problem is that the command will be hard-coded. Thanks for the insight, anyways.

**Comment by endreszabo** [comment]
Wouldn't the '--reloadcmd "service nginx reload" After issue/renew, it's used to reload the server.' be sufficient? Also pay attention to the return value of the `acme.sh` script itself :)

**Comment by RockyTV** [comment]
@endreszabo No, because that command will run only when it finished and succeeded in deploying the certificate ans key to the respective files. What I'm asking for is a command that can be executed when a deploy hook succeeds.

**Comment by trompx** [comment]
Any update on that?

From the doc:

- https://github.com/Neilpang/acme.sh/wiki/Options-and-Params
   `--reloadcmd "[command]" Command used after issue/renew, usually to reload the server.`

it is not mentioned it is only for `--install-cert`

I struggle to know if there is a hook that is triggered only once certificates have been either generated successfully or renewed successfully but not on attempt?

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.
